social_core/backends/auth0.py

```diff
@@ -0,0 +1,41 @@
+"""
+Auth0 implementation based on https://auth0.com/docs/quickstart/webapp/django/01-login
+"""
+from six.moves.urllib import request
+from jose import jwt
```

Comment:
This dependency should be added to the requirements in this package.

To do so you have to add `python-jose` to the project requirements.

https://github.com/raccoongang/social-core/blob/master/setup.py#L47
https://github.com/raccoongang/social-core/blob/master/requirements-base.txt

Reply:
Why do we ever need this library? It isn't enough to use `jwt`?
If it required to use this lib - add a link to the Auth0 documentation describing this intention.

Reply:
I have tried to use `jwt` but got errors while decoding.
`Jose` is also used in [auth0 tutorial](https://auth0.com/docs/quickstart/webapp/django/01-login).

Reply:
Another [topic](https://github.com/python-social-auth/social-core/pull/309) (this PR was taken as base for our backend). As written there: 

> in order to use any jwt-related backend, you need to install python-social-auth with the openidconnect extra:

`pip install social-auth-core[openidconnect]`
[requirements-openidconnect.txt](https://github.com/python-social-auth/social-core/blob/3ac1fb890cae460dc54abdef369cb9fa28409e53/requirements-openidconnect.txt) contains `python-jose`

Reply:
In our version of social-core `openidconnect` doesn't require `python-jose`, so I'll add it to requirements-base.txt